contentgrid-spring-integration-events/src/main/java/com/contentgrid/spring/integration/events/ContentGridPublisherEventListener.java

```diff
@@ -58,10 +59,11 @@ public boolean requiresPostCommitHandling(EntityPersister persister) {
 
     @Override
     public void onPostInsert(PostInsertEvent event) {
-        contentGridEventPublisher
-                .publish(new ContentGridMessage(applicationProperties.getSystem().getApplicationId(), applicationProperties.getSystem().getDeploymentId(),
-                        ContentGridMessageType.create, new DataEntity(null, event.getEntity()), event.getEntity().getClass(),
-                        Map.of("webhookConfigUrl", applicationProperties.getSystem().getWebhookConfigUrl())));
+        contentGridEventPublisher.publish(
+                new ContentGridMessage(applicationProperties.getSystem().getApplicationId(),
+                        applicationProperties.getSystem().getDeploymentId(),
+                        ContentGridMessageType.create, new DataEntity(null, event.getEntity()),
```

Comment:
For the next PR: enum values are constants, should be upper case, like: `ContentGridMessageType.CREATE` etc